modules/darwin/src/main/ts/ephox/darwin/mouse/MouseSelection.ts

```diff
@@ -46,7 +39,23 @@ export const MouseSelection = (bridge: WindowBridge, container: SugarElement, is
   };
 
   /* Keep this as lightweight as possible when we're not in a table selection, it runs constantly */
-  const mouseup = (_event?: EventArgs) => {
+  const mousedown = (event: EventArgs<MouseEvent>) => {
+    annotations.clear(container);
+    findCell(event.target, isRoot).each(cursor.set);
+  };
+
+  /* Keep this as lightweight as possible when we're not in a table selection, it runs constantly */
+  const mouseover = (event: EventArgs<MouseEvent>) => {
+    applySelection(event);
+  };
+
+  /* Keep this as lightweight as possible when we're not in a table selection, it runs constantly */
+  const mouseup = (event?: EventArgs<MouseEvent>) => {
```

Comment:
I was wondering if we should make this event mandatory. Thoughts?

Reply:
I don't know why it would be not mandatory — but that probably means I'm missing the original reason (on account of me not knowing very much at all about this code)

Reply:
I suspect it just got incorrectly converted when adding types since it's not currently used in the function :shrug:

Reply:
Have changed this to mandatory as discussed.